.azure-pipelines/scripts/docker-upload.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -ex
```

Comment:
Maybe We can change to use azure pipelines task instead.
https://docs.microsoft.com/en-us/azure/devops/pipelines/ecosystems/containers/push-image?view=azure-devops

- task: Docker@2
  displayName: Push image
  inputs:
    containerRegistry: |
      $(dockerHub)
    repository: $(imageName)
    command: push
    tags: |
      test1
      test2